rpc/server.go

```diff
@@ -51,20 +54,53 @@ func NewServer() *Server {
 	}
 }
 
+// NewApiServer creates a new Server.
+func NewApiServer(mods []api.Module, api *api.Api) *Server {
+	s := &Server{
+		services: new(serviceMap),
+		api: api,
+	}
+
+	s.RegisterModules(mods)
+
+	return s
+}
+
+// RegisterModules registers the RPC services associated with the given API modules
+func (s *Server) RegisterModules(mods []api.Module) {
+	for _, mod := range mods {
+		log.Debug("[rpc] Enabling rpc module", "module", mod)
+		var srvc interface{}
+		switch mod {
+		case "core":
```

Comment:
Rather than utilize the names of the same modules as [polkadotjs](https://polkadot.js.org/api/METHODS_RPC.html#json-rpc) even though there is nothing in the spec consistency is always better. I would suggest changing `core` to `system`.

Reply:
Renamed